### DISCUSSION TITLE
Item Delivery Service

### DISCUSSION THREAD

**Discussion by Jlobblet** [FakeFishGames/Barotrauma#6601] [Ideas and suggestions]
A frequently asked question is how to transfer items between submarines in campaign mode. This discussion proposes a solution to that problem.

The game should add an item delivery service. Accessible at outposts next to the submarine delivery service, players can deposit items into a GUI. Deposited items will be spawned as cargo in the next round. Player will have to either provide their own containers (i.e. metal crates, or a suitable alternative) or pay to buy them. This price stops steel from deconstructed crates being infinitely farmed.  
Alternatively, a new kind of crate could be added to the game, e.g. a "moving crate". It could have no deconstruction recipe, or deconstruct into nothing (as a disposal method) to avoid creating resources directly.

I think this provides an elegant way to solve a common question/feature request.

**Comment by James-E-A** [comment]
With the new update, you could even have the submarine delivery panel let you search your stowed subs for specific items to retrieve
